.gitignore

```diff
@@ -42,3 +42,4 @@ user.props
 # Generated rst files
 ######################
 genrst/
+
```

Comment:
Does this patch have any purpose? Should we let it go to have less diff against upstream master as it is outside the nrf/ folder?